dev-java/jrobin/jrobin-1.6.0-r2.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/OpenNMS/jrobin/archive/jrobin-1.6.0-1.tar.gz --slot 0 --keywords "~amd64 ~x86" --ebuild jrobin-1.6.0-r2.ebuild
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.jrobin:jrobin:1.6.0"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="JRobin is a 100% pure Java alternative to RRDTool"
+HOMEPAGE="https://github.com/OpenNMS/jrobin"
+SRC_URI="https://github.com/OpenNMS/${PN}/archive/${P}-1.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Common dependencies
+# POM: pom.xml
+# junit:junit:4.11 -> >=dev-java/junit-4.13.2:4
+
+CP_DEPEND="
+	dev-java/junit:4
```

Comment:
i think this is wrongly declared in `pom.xml` and should not be needed for runtime:
```
 * Messages for package dev-java/jrobin-1.6.0-r2:

 * Possibly unneeded dependencies found
 * 	junit-4
```